prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts

```diff
@@ -68,7 +68,8 @@ export class CollegeCertificationFormComponent implements OnInit {
   ) {
     this.remove = new EventEmitter<number>();
     // filter the college licenses that have been discontinued
-    this.licenses = this.configService.licenses.map(l => {
+    this.licenses = this.configService.licenses.map(x => Object.assign({}, x));
+    this.licenses.forEach(l => {
```

Comment:
Please add comment re: filtering out

Reply:
The comment is updated with more detail.